test/flow_map/reachable_region_algorithm.cpp

```diff
@@ -5,18 +5,13 @@
 
 #include <QApplication>
 
-#include "../../cartocrow/flow_map/painting.h"
-#include "../../cartocrow/flow_map/spiral_tree.h"
-#include "../../cartocrow/flow_map/spiral_tree_obstructed_algorithm.h"
-#include "../../cartocrow/renderer/geometry_painting.h"
-#include "../../cartocrow/renderer/geometry_widget.h"
-#include "../../cartocrow/renderer/ipe_renderer.h"
-#include "cartocrow/renderer/geometry_widget.h"
+#include "cartocrow/flow_map/reachable_region_algorithm.h"
```

Comment:
Include style. Actually this is probably better than the style in the other test files. So update those